src/components/CalculatorFilters.vue

```diff
@@ -29,14 +38,43 @@ export default defineComponent({
     },
     computed: {
         filters() {
-            this.calculatorStore.setFilters()
             return this.calculatorStore.activeFilters
         },
     },
     methods: {
         saveNewFilterValues() {
             this.calculatorStore.updateActiveFilters(this.filters)
         },
+        isSelected(group: string, option?: string): boolean {
+            if (!this.filters[group].size || this.filters[group].size < 1 || this.filters[group].size === this.calculatorStore.filterProperties[group].options.length) {
```

Comment:
`!this.filters[group].size` is equivalent `this.filters[group].size < 1`. You only need one of these checks, not both.

`this.filters[group].size === this.calculatorStore.filterProperties[group].options.length` will never be true due to the way `select()` is implemented. You can exclude this check.